src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -4712,5 +4712,47 @@ public C()
             Assert.Equal(PublicNullableAnnotation.NotAnnotated, typeInfo.Type.NullableAnnotation);
             Assert.Equal(PublicNullableFlowState.NotNull, typeInfo.Nullability.FlowState);
         }
+
+        [Fact, WorkItem(47467, "https://github.com/dotnet/roslyn/issues/47467")]
+        public void GetDeclaredSymbolTopLevelStatementsWithLocalFunctionFirst_1()
+        {
+            var comp = CreateCompilation(@"void M() {}", options: TestOptions.ReleaseExe.WithNullableContextOptions(NullableContextOptions.Enable));
```

Comment:
>@"void M() {}" [](start = 41, length = 14)

Could this be an `[InlineData()]` argument with the same test method shared across all three tests?